src/assignment/hooks/assignment-content-modals.tsx

```diff
@@ -48,9 +49,12 @@ export function useAssignmentContentModals(
 				block={getConfirmSliceModalBlock as AssignmentBlock}
 				onClose={() => setConfirmSliceModalOpen(false)}
 				onConfirm={() => {
-					const blocks = assignment.blocks.filter(
-						(item) => item.id !== getConfirmSliceModalBlock?.id
-					);
+					const blocks = assignment.blocks
+						.filter((item) => item.id !== getConfirmSliceModalBlock?.id)
+						.map((block, i) => ({
+							...block,
+							position: i,
+						}));
```

Comment:
We kunnen niet zeker zijn dat `assignment.blocks` op ieder moment een gesorteerde array is; Als we `position` gelijk willen houden aan de index voegen we best eenmalig sorting & mapping toe on-save.

Reply:
er is toch geen enkel moment waarbij we die volgorde breken?
zal het verwijderen, maar dan krijg je volgordes als 0,1,2,5,6,9,11,12

Reply:
updated

Reply:
De volgorde wordt gebroken wanneer er items worden tussengevoegd, opgeslagen en vervolgens verwijderd, dan krijg je inderdaad `0,1,2,5,6,9,11,12`.